samples/mqtt/native/README.md

```diff
@@ -15,6 +15,29 @@ It is possible to configure the example by creating and modifying a `config.toml
 During the sdk development several paramenters will be added to this file to extend the
 example functionalities.
 
+An example of `config.toml` file is the following:
+```
+[general]
+pairing_url = ""
+realm = ""
+device_id = ""
+
+[mode]
```

Comment:
Mode sounds like a choice of just one. While users can select multiple of those options at once. How about calling it sample features?